examples/11_particle_dataframe.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
Is it python 2 or python3?

Reply:
All python3, could specify here but that is ensured anyway via packaging logic.